### PR TITLE
Only recommend LineageOS official builds

### PR DESCRIPTION
**Pull request by jonaharagon**
image

### CODE REVIEW

File: _includes/sections/mobile-operating-systems.html [hunk 1]
```diff
@@ -16,9 +16,9 @@ <h1 id="mobile_os" class="anchor"><a href="#mobile_os"><i class="fas fa-link anc
 %}
 
 {% include cardv2.html
-  title="LineageOS"
+  title="LineageOS (Official Builds)"
   image="/assets/img/svg/3rd-party/lineageos.svg"
-  description='LineageOS is a free and open-source operating system for smartphones and tablets, based on the official releases of the Android Open Source Project. It is the continuation of the CyanogenMod project.'
+  description='LineageOS is the a free and open-source successor to the CyanogenMod project. They have <a href="https://wiki.lineageos.org/devices/">official support for a variety of devices</a>, based on clean releases of the Android Open Source Project. For security reasons, we do not recommend third-party builds of LineageOS for unsupported devices.'
```
Comment (nitrohorse) [inline _includes/sections/mobile-operating-systems.html:21]: Small typo with the added “the:” “the a free...”

Comment (nitrohorse) [inline _includes/sections/mobile-operating-systems.html:21]:
```suggestion
  description='LineageOS is a free and open-source successor to the CyanogenMod project. They have <a href="https://wiki.lineageos.org/devices/">official support for a variety of devices</a>, based on clean releases of the Android Open Source Project. For security reasons, we do not recommend third-party builds of LineageOS for unsupported devices.'
```

Comment (jonaharagon) [reply _includes/sections/mobile-operating-systems.html:21]: or... 
```suggestion
  description='LineageOS is the free and open-source successor to the CyanogenMod project. They have <a href="https://wiki.lineageos.org/devices/">official support for a variety of devices</a>, based on clean releases of the Android Open Source Project. For security reasons, we do not recommend third-party builds of LineageOS for unsupported devices.'
```

which is more accurate?

Comment (nitrohorse) [reply _includes/sections/mobile-operating-systems.html:21]: And good point, “the” makes more sense.

**Comment by koenvervloesem** [comment]
Someone who is new to the scene and doesn't know CyanogenMod (after all, the final release is already 3.5 years ago) still doesn't have a clue what LineageOS is after the first sentence. I found the old way with "It is the continuation of the CyanogenMod project." at the end clearer, so the most important information (what LineageOS is) comes first.

**Pull request merged**